### PR TITLE
Add Comic Potential

### PR DESCRIPTION
**Pull request by GingertronMk1**

### CODE REVIEW

File: _shows/18_19/comic_potential.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+- role: Music
+  name: Gordon Kershaw
+  person: false
+- role: Music
+  name: Giulia Grisot
+  person: false
+- role: Tech Box and Sound Effects
+  name: Lorenzo Bordin
+  person: false
+- role: Tech Box and Sound Effects
```
Comment (samozzy) [inline _shows/18_19/comic_potential.md]: These two are Technical Operators, with Lorenzo also a Lighting Designer. 
Zoe was also a Lighting Designer, and I wouldn’t say no to a Venue Technician role 😛

File: _shows/18_19/comic_potential.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+- role: Choreographer
+  name: Fabio Tanania
+  person: false
+- role: Choreographer
+  name: Eliana Fiorelli
+  person: false
+- role: Graphics
+  name: Davide Manna
+  person: false
+- role: Stage Design
```
Comment (samozzy) [inline _shows/18_19/comic_potential.md]: Set Design?

Comment (GingertronMk1) [reply _shows/18_19/comic_potential.md]: It's what was in the program

Comment (samozzy) [reply _shows/18_19/comic_potential.md]: Hmm, what is more important: truth to the programme, or consistency in role names? 
(Also: non-standard crew roles do not result in the nice icons appearing next to the roles) 
Not sure of answer myself.

Comment (GingertronMk1) [reply _shows/18_19/comic_potential.md]: If it's what they called the role...

We've come up with some stupid names for things in the past, idc if the postgrads are a bit off

**Pull request merged**